omod/src/main/java/org/openmrs/module/sync2/page/controller/Sync2PageController.java

```diff
@@ -0,0 +1,30 @@
+package org.openmrs.module.sync2.page.controller;
+
+import org.apache.commons.lang.StringUtils;
+import org.openmrs.module.sync2.api.utils.SyncUtils;
+import org.openmrs.module.uicommons.util.InfoErrorMessageUtil;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.page.PageModel;
+import org.springframework.stereotype.Controller;
+
+import javax.servlet.http.HttpSession;
+
+@Controller
+public class Sync2PageController {
+
+	private static final String VALID_PARENT_INSTANCE_URI = "validParentInstanceUri";
+
+	private static final String PARENT_URL_ERROR = "sync2.configuration.parentUrl.notValid";
```

Comment:
do we have constant class for message properties ?

Reply:
We have the file which contains constant variables but I we haven't there any message property key.